content/participants/ttjc@mitrish/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Rishabh Mittal
+type: "participant"
+name: "Rishabh Mittal"
+portfolio: "http://rishabhmittal.tech"
+github-link: "https://github.com/mitrish"
+linkedin-link: "https://linkedin.com/in/mittalrish"
+looking-for: "job"
+description: "Full Stack Web Developer Intern at Kickdrum India, Self learner, learning everyday"
+---
+
+Hi! I am Rishabh Mittal, let me walk you through my intro, projects and blogs.
+
+### introduction
+
+I have solid fundamentals (HTML,CSS,JS,SQL, MongoDB, Node) and have worked with frameworks such as React, Angular, SpringBoot, working towards AWS Developer Associate. Currently Focused on learning java and improving existing skill sets.
+
+## why are you taking this challenge?
+
+I need an way where i can continue to sharpen my skills, through this challenge i can keep track of my progress.
+
+## something interesting about you?
+
+Interested in the stock market, love playing chess.
+
+### help I need?
+
+Will keep updating if stuck somewhere.
+
+### how to contact me?
+
+- [Twitter](https://twitter.com/mittrish_01)
+- [LinkedIn](https://www.linkedin.com/in/mittalrish/)
+
+### projects
+
+Starting from scratch, please observe this space for updates.
+
+
+### blogs
+
+These are some of the blogs that i have already written in the past.
```

Comment:
You have added blogs which were done before jobChallenge. Only blogs/projects done during the challenge are accepted. Kindly remove them.